library/Haste/Model/Relations.php

```diff
@@ -855,18 +859,18 @@ public static function getRelation($strTable, $strField)
 
                     // Related table data
                     $varRelation['related_table'] = $arrField['table'];
-                    $varRelation['related_tableSql'] = $arrField['tableSql'];
+                    $varRelation['related_tableSql'] = $arrField['tableSql'] ?? null;
```

Comment:
I think we would need to raise the minimum PHP version (which is fine to me!) because `??` might not be supported by PHP 7.0?

Reply:
The null coalescing operator should be available in PHP 7.0: https://www.php.net/manual/en/migration70.new-features.php

Reply:
In https://github.com/codefog/contao-haste/pull/160 @m-vo allready changed the requirement to `"php": "~7.0 || ~8.0"`
But I was not sure about the prefered syntax - see: https://github.com/codefog/contao-haste/pull/160#pullrequestreview-595120022
Allthough I would prefer the coalescing operator!